Tests/test_image_transform.py

```diff
@@ -178,11 +178,13 @@ def _test_nearest(self, op, mode):
 
         im = op(im, (40, 10))
 
-        colors = im.getcolors()
-        assert colors == [
-            (20 * 10, opaque),
-            (20 * 10, transparent),
-        ]
+        colors = sorted(im.getcolors())
+        assert colors == sorted(
+            (
+                (20 * 10, opaque),
+                (20 * 10, transparent),
+            )
+        )
```

Comment:
What is your thinking here? I know we've had this conversation in https://github.com/python-pillow/Pillow/pull/6540#discussion_r979468526 already. Are you trying to pave the way for that PR?

Reply:
I'm making this test match this other test:
https://github.com/python-pillow/Pillow/blob/bf00a4534d728dff93232142cf99aabe4e8174fd/Tests/test_image.py#L313-L342

Reply:
Ok, I see that `getcolors()` is only guaranteed to return an "[unsorted list](https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.Image.getcolors)", and the test in test_image.py actually returns a different order on big-endian.